stl/inc/yvals_core.h

```diff
@@ -1007,7 +1007,19 @@
 #define _CXX20_DEPRECATE_MOVE_ITERATOR_ARROW
 #endif // ^^^ warning disabled ^^^
 
-// next warning number: STL4032
+#if _HAS_CXX20 && !defined(_SILENCE_CXX20_POLYMORPHIC_ALLOCATOR_DESTROY_DEPRECATION_WARNING) \

```

Comment:
This deprecation needs to be active in C++17 mode: we don't want people to start writing code that uses `polymorphic_allocator::destroy` after applying this PR that deprecates it.